src/components/Image/index.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import { PureComponent, createElement } from 'react';
+import type MapboxMap from 'mapbox-gl/src/ui/map';
+import type { StyleImageInterface } from 'mapbox-gl/src/style/style_image';
+
+import MapContext from '../MapContext';
+
+type Props = {|
+  /** The ID of the image. */
+  id: string,
+
+  /**
+   * The image as an `HTMLImageElement`, `ImageData`, object with `width`,
+   * `height`, and `data`
+   * properties with the same format as `ImageData` or string with image url
+   * */
+  image:
+    | window.HTMLImageElement
```

Comment:
One should not use `window` in type definitions

Reply:
I can use mapbox types like they do it [here](https://github.com/mapbox/mapbox-gl-js/blob/master/src/ui/map.js#L6). Is it ok?